cmake/Config.cmake

```diff
@@ -50,14 +50,6 @@ elseif(${CMAKE_SYSTEM_NAME} STREQUAL "Darwin")
 
         # don't use the OpenGL ES implementation on Mac OS X
         set(OPENGL_ES 0)
-
-        # detect OS X version. (use '/usr/bin/sw_vers -productVersion' to extract V from '10.V.x'.)
-        EXEC_PROGRAM(/usr/bin/sw_vers ARGS -productVersion OUTPUT_VARIABLE MACOSX_VERSION_RAW)
-        STRING(REGEX REPLACE "10\\.([0-9]+).*" "\\1" MACOSX_VERSION "${MACOSX_VERSION_RAW}")
-        if(${MACOSX_VERSION} LESS 7)
-            message(FATAL_ERROR "Unsupported version of OS X: ${MACOSX_VERSION_RAW}")
-            return()
-        endif()
```

Comment:
Is the requirement now given through other incompatibilities?

Reply:
That's right. Here's the extended commit message where I go into more detail:

> [Remove check for old macOS versions](https://github.com/SFML/SFML/pull/2178/commits/cae9733067814f901efb1f7596eb5fb198e73a2d)  …
There is no need to check for macOS 10.6 or lower. Snow Leopard came
out in 2009. Because it's nearly 13 years old, it's safe to say
nobody will be trying to build SFML 3 on Snow Leopard.
This was added in https://github.com/ChrisThrasher/SFML/commit/de70f691e10b502e09d1683b2d241d559f199948 way back in 2011 when Snow Leopard was
a mere cub.